cclib/parser/gaussianparser.py

```diff
@@ -2099,5 +2103,15 @@ def natural_orbital_single_spin_parsing(inputfile, updateprogress_title):
                     self.optdone = []
                 self.optdone.append(len(self.optstatus) - 1)
 
+        # Extract total elapsed and CPU job times in seconds
+        if line[:14] == ' Elapsed time:':
+            days, hours, minutes, seconds = float(line.split()[2]), float(line.split()[4]), float(line.split()[6]), float(line.split()[8])
```

Comment:
Can you change this to only `line.split()` once into a separate variable, then use that?